File: 242-valid-anagram/242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
    
      char string1[] = s.toCharArray();
      char string2[] = t.toCharArray();
        
        
        
      Arrays.sort(string1);
      Arrays.sort(string2);
        
        String comp1 = new String(string1);
        String comp2 = new String(string2);
       return comp1.equals(comp2);

    
    }
}

File: 242-valid-anagram/242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        lists = {}
        listt = {}
        
        for letter in s:
            lists[letter] = lists.get(letter,0) + 1
            
        for letter in t:
            listt[letter] = listt.get(letter, 0) + 1
       
        print(listt)
        print(lists)
        
        return listt == lists
        